GOVERNANCE.md

```diff
@@ -197,6 +197,21 @@ Admins are responsible for:
 - Being the point of contact with the CNCF (the CNCF calls this group "maintainers", see [here](https://github.com/cncf/foundation/blob/main/project-maintainers.csv))
 
 Some administrative responsibilities are handled by contributors that do not meet the typical maintainer requirements with regard to code review and contributions.  These contributors are nonetheless recognized and extended admin access as is necessary to fulfill the needs of the community including infrastructure, operational, security, and billing functions.  These admins may require, for example, GitHub owner access, access to credentials, and CNCF maintainers list access.
+
+---
+
+## Role Fast Tracking
+
+In certain circumstances it may be appropriate to fast track individuals into elevated roles. 
+
+An individual to be fast tracked _should_ satisfy one or more of the following criteria:
+
+- Raised a single significant contribution that positively effects the project as determined by existing maintainers.
+- Has demonstratable knowledge in closely related projects that translates to the target project. For example, Kubernetes controller knowledge may translate between projects.
+- Meets all the requirements of roles with lesser responsibilities. For example, a fast track to Approver may occur if an individual meets all the Reviewer and Approver requirements in a short space of time (typically less than the timelines outlined for each role).
+
+An individual to be fast tracked _must_ have expressed support from a majority of existing maintainers.
```

Comment:
That sound sensible to me else there's typically just 2-3 people involved and, in theory, the process could be abused.

Reply:
Perhaps something like 

> An individual under consideration _must_ have expressed support from a majority of Tinkerbell maintainers.

Reply:
A majority of tinkerbell maintainers is going to be very difficult to gather as we scale. 
A fixed number would be practical. While this could also be abused with a clique, appointing new members, I think we have to trust this will be done with good intention.

Perhaps we can look at this from the opposite direction: "without objection after n days after public notice via (mailing list?)".

Reply:
I like that this is worded in a way that doesn't preclude fast-tracking admins.

Reply:
Perhaps the majority of the target project, 1 admin and 1 maintainer not in the project. The goal here is to try and bring about some sort of sanity check that existing maintainers aren't just throwing people in?

Reply:
LGTM.

DCO to resolve.